modules/alacritty/hm.nix

```diff
@@ -1,31 +1,60 @@
-{pkgs, config, lib, ... }:
-
-with config.stylix.fonts;
+# Documentation is available at https://alacritty.org/config-alacritty.html.
```

Comment:
AFAIK, it is fairly rare in the Nix ecosystem to specify an upstream reference. This could be due to laziness, reduced maintenance of upstream references, or finding upstream documentation being considered trivial.

I have no strong feelings about this. @danth, what do you think?

Reply:
I don't see any reason not to include this - it saves the time of going looking for it later.